Tools/GenerateWebsiteData.ps1

```diff
@@ -8,6 +8,8 @@ Param(
     [string]$configuration = "Release"
 )
 
+$script:defaultParameterSetName = "Default"
```

Comment:
Do we need this? It only seems to be used in one place, and there it is just read?